helmchart/sdp/README.md

```diff
@@ -121,8 +121,8 @@ Configuration parameters for the SDP Core Service which is the core backend serv
 | `sdp.configMap.data.SMS_SENDER_TYPE`                       | The messenger type used to send text messages to recipients. Options: "DRY_RUN", "TWILIO_SMS".                                                                 | `DRY_RUN`                                       |
 | `sdp.configMap.data.RECAPTCHA_SITE_KEY`                    | Site key for ReCaptcha. Required if using ReCaptcha.                                                                                                           | `nil`                                           |
 | `sdp.configMap.data.CORS_ALLOWED_ORIGINS`                  | Specifies the domains allowed to make cross-origin requests. "*" means all domains are allowed.                                                                | `*`                                             |
-| `sdp.configMap.data.ENABLE_RECAPTCHA`                      | Determines if ReCaptcha should be enabled for login ("true" or "false").                                                                                       | `false`                                         |
-| `sdp.configMap.data.ENABLE_MFA`                            | Determines if email-based MFA should be enabled during login ("true" or "false").                                                                              | `false`                                         |
+| `sdp.configMap.data.DISABLE_RECAPTCHA`                     | Determines if ReCaptcha should be disabled for login ("true" or "false").                                                                                      | `false`                                         |
+| `sdp.configMap.data.DISABLE_MFA`                           | Determines if email-based MFA should be disabled during login ("true" or "false").                                                                             | `false`                                         |
```

Comment:
We'll need to publish a new chart probably for this change ? https://github.com/stellar/helm-charts

Reply:
Thanks for catching that! PR opened in https://github.com/stellar/helm-charts/pull/77.